src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -130,24 +130,25 @@ bool convert_precision(ov::pass::PassBase& pass,
     };
 
     auto convert_node_output_precision = [&](const std::shared_ptr<ngraph::Node>& node) {
+        bool res = false;
         for (const auto& output : node->outputs()) {
             if (output.get_element_type() == from) {
                 // Handle case with Constants as they can have consumers from other nGraph
                 // Function object
                 auto it = const_to_internal_output.find(node.get());
                 if (it != const_to_internal_output.end()) {
-                    return fuse_type_to_constant(node, to, it->second);
+                    res |= fuse_type_to_constant(node, to, it->second);
+                    continue;
```

Comment:
It seems that `fuse_type_to_constant` is performed "per node" not "per output", 
Does it make sense to call `continue` instead of `break` or immediate `return` as it was originally done?

Reply:
I believe that `fuse_type_to_constant` will be applied if `node` is a constant. 
Constant has only one output so it won't fail.
But it's better to move it before `for` loop`, e.g.
```
auto const = ov::as_type_ptr<opset10::Constant>(node);
if (const && const_to_internal_output.find(node.get())) {
   return fuse_type_to_constant(const /* looks like we can change the type from Node to Const */, to, it->second);;
}
```